### PR TITLE
[Resolves #1] project boilerplate

### LINKED ISSUE #1: Setup Django project boilerplate
*AC:*
* Setup basic Django app with "Hello world" view.
* Add basic Makefile.
* Add README how to start the app.

### PR DESCRIPTION
**Pull request by stanislawK**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+wait-for-it "${DB_HOST}:${DB_PORT}"
```
Comment (maciejsobolewski) [inline docker-entrypoint.sh]: It's an ok solution, but you can use a `healthcheck` in docker-compose.
See: https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+RUN pip install --upgrade pip \
+    && pip install poetry==1.1.13
+
+COPY docker-entrypoint.sh poetry.lock pyproject.toml /
+
+RUN chmod +x /docker-entrypoint.sh
+RUN poetry config virtualenvs.create false \
+    && poetry install --no-interaction --no-ansi
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
```
Comment (maciejsobolewski) [inline Dockerfile]: You can specify entrypoint in docker-compose.

Comment (maciejsobolewski) [inline Dockerfile]: You might still want to copy the files to the container at the end of the Dockerfile.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
 # Quiz
+
+Brand new quiz app
+
+## Development environment
+
+We use Docker for development purposes. To start working, your computer needs to support it.
```
Comment (maciejsobolewski) [inline README.md]: Docker + docker-compose. You might add a link on how to install them.

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+services:
+    db:
+        image: postgres:14
+        container_name: quiz_app_db
+        environment: 
+            - POSTGRES_DB=quiz_app
+            - POSTGRES_USER=quiz_app
+            - POSTGRES_PASSWORD=secret
```
Comment (maciejsobolewski) [inline docker-compose.yml:10]: I would add postgres db-data as a volume to persist data.

**Review by maciejsobolewski — CHANGES_REQUESTED** [review]

**Review by maciejsobolewski — CHANGES_REQUESTED** [review]
See my comments.

File: Dockerfile (hunk 2 above)
Comment (stanislawK) [reply Dockerfile]: I followed suggestions from [that](https://stackoverflow.com/questions/53835198/integrating-python-poetry-with-docker) post's "caching dependencies" part.

Comment (maciejsobolewski) [reply Dockerfile]: Yup, and it's a good practice, but in the end you still need to copy the code.
I just saw that you're doing that in the line before: `COPY ./quiz_app /quiz_app` - my bad.

**Review by maciejsobolewski — APPROVED** [review]

**Pull request merged**